### PR TITLE
feat: ESC key selects parent of selected node

### PR DESCRIPTION
**Pull request by luca-domenichini**
This PR implements #388.
Pressing ESC key selects the parent of the currently selected node.

### CODE REVIEW

**Comment by mimmoz81** [comment]
@abhinayagarwal any news on this?

File: kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/hierarchy/treeview/HierarchyTreeViewController.java [hunk 1]
```diff
@@ -85,6 +88,22 @@ protected void initializePanel() {
         // We do not use the platform editing feature because 
         // editing is started on selection + simple click instead of double click
         treeView.setEditable(false);
+        
+        treeView.setOnKeyPressed(event -> {
```
Comment (AlmasB) [inline kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/hierarchy/treeview/HierarchyTreeViewController.java:92]: Is there any reason for why this is `setOnKeyPressed()` and the other change is `addEventFilter()`?

**Review by AlmasB — COMMENTED** [review]
Looks good, thanks. Please see below

File: kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/hierarchy/treeview/HierarchyTreeViewController.java (hunk 1 above)
Comment (mimmoz81) [reply kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/hierarchy/treeview/HierarchyTreeViewController.java:92]: Hello, sorry, I did not receive any mail about your comment. Let me check in the next days.

Comment (mimmoz81) [reply kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/hierarchy/treeview/HierarchyTreeViewController.java:92]: Changed to always call `setOnKerPressed()`

Comment (Oliver-Loeffler) [reply kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/hierarchy/treeview/HierarchyTreeViewController.java:92]: Technically this looks okay for me.
But the license header requires an update. It requires the line ` * Copyright (c) 2022 Gluon and/or its affiliates.` as first line.

**Review by Oliver-Loeffler — CHANGES_REQUESTED** [review]
Please update first license header line in in `ContentPanelController.java` from `* Copyright (c) 2016, 2018 Gluon and/or its affiliates.` to `* Copyright (c) 2016, 2022 Gluon and/or its affiliates.`.

Also please add the line `* Copyright (c) 2022 Gluon and/or its affiliates.` as new first line in `HierarchyTreeViewController.java`.

**Comment by Oliver-Loeffler** [comment]
Hi @luca-domenichini ,
sorry that the review really took that long. Could you please update the license headers of both modified files?
Thanks for your contribution - I've played with it and it works fine.

**Comment by luca-domenichini** [comment]
@Oliver-Loeffler done

**Review by Oliver-Loeffler — APPROVED** [review]

**Pull request merged**